fluent-syntax/test/fixtures_behavior/placeable_in_placeable.ftl

```diff
@@ -8,7 +8,7 @@ key2 = {  { foo }  }
 #   }
 
 key4 = {  { foo }
-# ~ERROR E0003, pos 93, args "}"
+# ~ERROR E0003, pos 96, args "}"
```

Comment:
This is weird. Why is `#` OK in a placeable?

Reply:
It's not. The error is exactly on that character:

```
! E0003 on line 2:
  | key4 = {  { foo }
  | # ~ERROR E0003, pos 93, args "}"
  … ^----- Expected token: "}"
```

Reply:
Ok, so we dove into this with Axel. The reason this annotation stops at 96 now, is that the test infra removes the `# ~ERROR` comment, so the parser continues and stops on the first character of `# key5` two `\n`'s later which is 96 :)